src/solve.jl

```diff
@@ -1,8 +1,9 @@
 function init(prob::AbstractDDEProblem{uType,tType,lType,isinplace}, alg::algType,
               timeseries_init=uType[], ts_init=tType[], ks_init=[];
               d_discontinuities=tType[], dtmax=tType(7*minimum(prob.lags)), dt=zero(tType),
-              kwargs...) where {uType,tType,isinplace,
-                                algType<:AbstractMethodOfStepsAlgorithm,lType}
+              saveat=tType[], tstops=tType[], save_idxs=nothing,
```

Comment:
Since the internal solution of the ODE integrator is supposed to be always a full solution with all components that saves every step. The solution of the DDE integrator can be modified by these keywords.